src/test/util.test.ts

```diff
@@ -0,0 +1,48 @@
+import { compareFloats } from "../util";
+
+describe("util", () => {
+    describe("compareFloats", () => {
```

Comment:
Thank you!! Great job adding a unit test.

Reply:
😄 It was necessary for my own sanity